query/src/main/scala/filodb/query/exec/aggregator/AvgRowAggregator.scala

```diff
@@ -51,6 +51,6 @@ object AvgRowAggregator extends RowAggregator {
   }
   def presentationSchema(reductionSchema: ResultSchema): ResultSchema = {
     // drop last column with count
-    reductionSchema.copy(columns = reductionSchema.columns.take(reductionSchema.columns.size-1))
+    reductionSchema.copy(reductionSchema.columns.filterNot(_.name.equals("count")))
```

Comment:
Consider using `find` or `filter` with required columns instead of `filterNot` - more readable. Same with other related changes.